deepchem/models/torch_models/chemnet_layers.py

```diff
@@ -0,0 +1,51 @@
+"""
+Contains implementations of layers used in ChemCeption .
+"""
+import torch
+import torch.nn as nn
+
+
+class Stem(nn.Module):
+    """
+    Implements Stem Layer as defined in https://arxiv.org/abs/1710.02238.
+    This layer downsamples the image to reduce computational complexity before passing it to deeper layers.
+
```

Comment:
Can you please add parameters to docstrings? Also, I think including more backgorund about this layer will be useful.

Reply:
> Can you please add parameters to docstrings? Also, I think including more backgorund about this layer will be useful.

`Can you please add parameters to docstrings?` This is for forward method